python/lsst/ts/wep/task/reassignCwfsCutoutsTask.py

```diff
@@ -70,7 +72,7 @@ def adjust_all_quanta(self, adjuster):
                 seen.add(data_id)
             elif data_id["detector"] in intra_focal_ids:
                 extra_focal_data_id = DataCoordinate.standardize(
-                    data_id, detector=data_id["detector"] - 1
+                    data_id, detector=int(data_id["detector"]) - 1
```

Comment:
Isn't `dataId['detector']` already an `int`?

Reply:
I think that it's because even though you know data_id["detector"] is already an int, mypy can't infer this, especially when dealing with dictionary lookups, because from its perspective, dictionary values can have arbitrary types unless explicitly typed.